src/Inquirer/Validators/RangeValidator.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+
+namespace InquirerCore.Validators
+{
+  public class RangeValidator : IValidator
+  {
+    private int Min { set; get; }
+    private int Max { set; get; }
+
+    public RangeValidator(int minValue, int maxValue)
+    {
+      Min = minValue;
+      Max = maxValue;
+    }
+
+    public bool Validate(string value)
+    {
+      try
+      {
+        int iVal = int.Parse(value);
```

Comment:
here also we don't need to use Hungarian notation.

Reply:
ok